### PR TITLE
init_stack64_dts9_patcherv

### PR DESCRIPTION
**Pull request by brandonros**

### CODE REVIEW

**Comment by brandonros** [comment]
in scemu (not libscemu) in cargo.toml i am putting this

`libscemu = { git = "https://github.com/brandonros/libscemu.git", branch = "master" }`

up to you

File: src/emu.rs [hunk 1]
```diff
@@ -249,7 +256,8 @@ impl Emu {
             self.maps.is_64bits = true;
             self.init_regs_dts9_patcherv();
```
Comment (brandonros) [inline src/emu.rs:257]: if you want to put these both behind an if flag, would be cool for others (if any)

**Comment by sha0coder** [comment]
>>libscemu = { git = "https://github.com/brandonros/libscemu.git", branch = "master" }

I don't understand this, the git that feeds the crates.io is not this, could be confusing,  and this pull request is not in scemu,

but I was thinking add you as co-author if you want, since you are improving the quality of this software a lot.

**Comment by sha0coder** [comment]
Do you have telegram? I have created a SCEMU group

**Comment by brandonros** [comment]
yes, what is your telegram?

**Pull request merged**